ex04.ex

```diff
@@ -39,7 +39,9 @@ defmodule Ex04 do
       [ 1, 2, 3, 4, 5 ]
 
   """
-  def reverse . . . "your code"
+  def reverse([]), do: []
```

Comment:
# -1

There's no need to test for an empty list, because reduce already handles it.